src/script/components/LegalHoldDot.tsx

```diff
@@ -49,14 +48,18 @@ const LegalHoldDot: React.FC<LegalHoldDotProps> = ({
   className = '',
   dataUieName = 'legal-hold-dot-pending-icon',
 }) => {
+  const {showRequestModal, showUsers} = useAppMainState(state => state.legalHoldModal);
+
   const onClick = (event: React.MouseEvent) => {
     event.stopPropagation();
 
-    if (isPending) {
-      amplify.publish(LegalHoldModalState.SHOW_REQUEST);
+    if (!isPending) {
+      showRequestModal(false, true);
+
+      return;
     }
 
-    amplify.publish(LegalHoldModalState.SHOW_DETAILS, conversation);
+    showUsers(false, conversation);
```

Comment:
The logic seem to have changed there, why do we have an early return now that was not there before?

Reply:
also the condition changed from `pending` to `!pending`  🤔

Reply:
No we don't!!!
![image](https://user-images.githubusercontent.com/1090716/196209119-cd90d5ff-6da9-4053-8ca3-e727e57f6eb6.png)